static/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+group1:
```

Comment:
Hmm... I suppose we will want all the folders for data (i.e. static and dynamic) in a testing folder, since they aren't technically part of the actual program. 

Until we get basic tests worked out, this works.